SPECS/xapi.spec

```diff
@@ -436,6 +437,9 @@ Coverage files from unit tests
 %endif
 
 %changelog
+* Tue Jun 25 2019 Benjamin Reis <benjamin.reis@vates.fr> - 1.160.1-1.2
```

Comment:
Changing the release number here is not enough, you need to change it in the Release: tag above too.